docs/ConfigurationFile.md

```diff
@@ -106,6 +106,14 @@ exports.config = {
           // flag to activate Firefox headless mode (see https://github.com/mozilla/geckodriver/blob/master/README.md#firefox-capabilities for more details about moz:firefoxOptions)
           // args: ['-headless']
         }
+    }, {
+        // If outputDir is provided WebdriverIO can capture driver session logs
+        // it is possible to configure which logTypes to exclude.
+        // excludeDriverLogs: ['*'], // pass '*' to exclude all driver session logs
+        excludeDriverLogs: ['bugreport', 'server'],
```

Comment:
Can you just incorporate this into the 2 capabilities above?

Reply:
I'm sorry, what do you mean to be honest.

excludeDriverLogs is only for driverlogs. 

If you want to disable particular logger just set it level to silent

Reply:
I thought `excludeDriverLogs` would exclude the log types being fetched after the test by https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-runner/src/index.js#L195-L241 , no?

Reply:
Yes (if we talking about same things).
We are getting logs supported by browser 
`        const logTypes = await global.browser.getLogTypes()`
then we are filtering them with excludeDriverLogs.

It has nothing to do with wdio loggers

Reply:
I was just saying instead of adding a new capability to this demo config , let's just add it to one of the existing ones.

Reply:
I'll update both doc and template and let you know